test/components/GlobalNavigasjonTest.tsx

```diff
@@ -294,6 +294,41 @@ describe("GlobalNavigasjon", () => {
       .to.exist;
   });
 
+  it('viser en rød prikk for menypunkt "Snart slutt på sykepengene" når kandidat varslet for minst ti dager siden uten svar', () => {
+    mockUnleashWithFeatureToggles();
+    queryClient.setQueryData(
+      senOppfolgingKandidatQueryKeys.senOppfolgingKandidat(fnr),
+      () => [
+        {
+          ...senOppfolgingKandidatMock,
+          varselAt: addDays(new Date(), -10),
+          svar: undefined,
+        },
+      ]
+    );
+    renderGlobalNavigasjon();
+
+    expect(screen.getByRole("link", { name: "Snart slutt på sykepengene 1" }))
+      .to.exist;
+  });
+
+  it('viser ikke en rød prikk for menypunkt "Snart slutt på sykepengene" når kandidat varslet i dag uten svar', () => {
+    mockUnleashWithFeatureToggles();
+    queryClient.setQueryData(
+      senOppfolgingKandidatQueryKeys.senOppfolgingKandidat(fnr),
+      () => [
+        {
+          ...senOppfolgingKandidatMock,
+          svar: undefined,
```

Comment:
Kunne man skrevet eksplisitt hva varselAt er her? For alt vi vet er varselAt undefined, da ville testen fortsatt blitt grønn siden fanen vises.

Reply:
Ja, enig, kan gjøre det mer eksplisitt i testene :+1: